source/documents/modify-multiple.txt

```diff
@@ -14,4 +14,146 @@ Modify Multiple Documents
+the :guilabel:`Update Documents` modal helps you visualize updates 
+to your data before you apply them.
+
+About this Task
+---------------
+
+- You can use any syntax that works with the ``update`` parameter of
+  :method:`db.collection.updateMany()`. 
+  
+- The :guilabel:`Update Documents` modal does not support any ``options`` 
+  parameters such as upsert, writeConcern, or collation.
+
+Before you Begin
+----------------
+
+The :guilabel:`Update Documents` modal is available starting
+in Compass version ``1.5.X``. For instructions on updating or installing
+the latest Compass version, see :ref:`download-install`. 
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Open the bulk update modal
+
+      From the :guilabel:`Documents` tab, click the :icon-lg:`Edit` 
+      :guilabel:`Update` button to display the 
+      :guilabel:`Update Documents` modal. The following 
+      table summarizes the UI of the modal:
+
+      .. list-table::
+         :widths: 40 60
+         :header-rows: 1
+
+         * - UI Element
+           - Description
+
+         * - :guilabel:`Filter`
+           - Any filter criteria specified on the 
+             :guilabel:`Query Bar` applies to the 
+             :guilabel:`Update Documents` modal.
+         * - :guilabel:`Update`
+           - The update syntax that is applied to the 
+             documents specified in the filter criteria. You can use 
+             any syntax that works with the ``update`` parameter of the
+             :method:`db.collection.updateMany()`.
+         * - :guilabel:`Preview`
+           - A preview of documents with the update syntax applied.
+
+   .. step:: Enter the update syntax 
+
+      In the :guilabel:`Update` text field, provide the update 
+      syntax.
+
+      The number of documents affected by the update displays in 
+      the top of the :guilabel:`Update Documents` modal in 
+      parentheses.
+
+      .. note::
+
+         The documents under the :guilabel:`Preview` header show how the 
+         :guilabel:`Update` syntax affects documents in your collection.
+
+   .. step:: (Optional) Name and save your update syntax
+
+      You can name and save the update query in the 
+      :guilabel:`Update Documents` modal. Saving your query adds it 
+      to your :ref:`favorite queries <saved-favorite-query-view>` for 
+      that collection and allows you to load and copy the query after 
+      you close the modal.
+
+      a. Click the :guilabel:`Save` button on the bottom left of the 
+         modal.
+      #. Enter a name for the update syntax.
+      #. Click the green :guilabel:`Save` button. A pop-up toast 
+         message displays, indicating the statement saved.
+
+   .. step:: Update your documents
+
+      Click :guilabel:`Update Documents`.
+
+      Compass applies the :guilabel:`Update` syntax to the documents
+      within the filter criteria in the :guilabel:`Filter` expression.
+
+Example
```

Comment:
At some point in this section, would it make sense to show an example before and after document? I'll leave it up to you to decide but it felt natural to me as a reader since you mentioned the preview section